tests/integration/s3/test_s3.py

```diff
@@ -25,7 +25,7 @@
 from botocore import UNSIGNED
 from botocore.auth import SigV4Auth
 from botocore.client import Config
-from botocore.exceptions import ClientError
+from botocore.exceptions import ClientError, ParamValidationError
```

Comment:
This kind of `ParamValidationError` are actually a client side validation made by boto. It does not make much sense to snapshot validate them, as they might be different for every SDK, and we only aim to support server side validation. If you want to validate the server side validation, you will need to disable the client side one.
Here are some link on how to do it: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html see `parameter_validation `. We also have an example in code, here: https://github.com/localstack/localstack/blob/master/localstack/aws/forwarder.py#L163-L167

Reply:
Sorry, my bad ! I've updated the tests.